_includes/assets/js/inline.js

```diff
@@ -22,5 +22,17 @@ $(document).ready(function(){
 			$('button.header-main__menu').attr("aria-expanded","true");
 		}
 	});
+
+	// Slick https://kenwheeler.github.io/slick/ - Home page carousel
+	$('.carousel-block__home').slick({
+		dots: true,
+		arrows: true,
```

Comment:
Are we having arrows as well as dots? Just comparing the designs, happy to wait until Lyndsay gets back before we make any changes

Reply:
I've turned arrows on for now yeah, to be extra accessible, but thet can easily be turned off.

1px gap removed, dots increased size, active dot is now orange. This matches designs but could be worth talking to Lyndsay when she's back about the size and colour.

Reply:
@lewisnyman I'm going to merge this in so I can crack on with streams of work that are dependant on this for styling. I've kept a note of the above so I know that the inclusion of the arrows and their styles do still need discussing.